db/flags.sql

```diff
@@ -7,9 +7,21 @@ CREATE TABLE IF NOT EXISTS flags.kvdata (
     corrected REAL NULL,
     controlinfo TEXT NULL,
     useinfo TEXT NULL,
+    -- TODO: check that this type is correct, it's stored as a string in Kvalobs?
```

Comment:
This is correct. Arguably something like `smallint[]` would have made more sense, but that's not what they chose 🤷‍♀️, and here we're just trying to be compatible

Reply:
Ok, then I probably should also use the same type in `old_flags`